### PR TITLE
[C-2227] Clear wallet state on sign-out

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes issue where signing out and logging in with new user gives the wallet address as previous user. This is dangerous and also has a lot of negative effects like showing incorrect balance and connected wallets

### CODE REVIEW

File: packages/common/src/store/pages/token-dashboard/slice.ts [hunk 1]
```diff
@@ -293,6 +293,12 @@ const slice = createSlice({
     },
     resetRemovedStatus: (state) => {
       state.associatedWallets.removeWallet.status = null
+    },
+    resetState: (state) => {
+      state.associatedWallets = initialState.associatedWallets
```
Comment (sliptype) [inline packages/common/src/store/pages/token-dashboard/slice.ts:298]: Is there are reason for only resetting parts of this slice? Why not set back to the initialState

**Review by sliptype — APPROVED** [review]

File: packages/common/src/store/pages/token-dashboard/slice.ts (hunk 1 above)
Comment (dylanjeffers) [reply packages/common/src/store/pages/token-dashboard/slice.ts:298]: for some weird reason doing `state = initialState` doesn't seem to work...

Comment (sliptype) [reply packages/common/src/store/pages/token-dashboard/slice.ts:298]: I've seen this weirdness with immer before, can you try returning initial state from this reducer?

Comment (dylanjeffers) [reply packages/common/src/store/pages/token-dashboard/slice.ts:298]: great idea, will give that a go

**Pull request merged**